onnxruntime/core/framework/execution_frame.cc

```diff
@@ -337,7 +337,7 @@ Status ExecutionFrame::AllocateMLValueTensorPreAllocateBuffer(OrtValue& ort_valu
 
     // be generous and use the buffer if it's large enough. log a warning though as it indicates a bad model
```

Comment:
@yufenglee the comment says that could be a bad model. Can you look into that as well?

Reply:
The root cause is that shape inference for Slice with dim_param is not correct, but ORT also has this logging issue. This is the fix of ORT side.